app/src/main/java/org/oppia/app/profile/AdminPinViewModel.kt

```diff
@@ -10,4 +10,6 @@ import javax.inject.Inject
 class AdminPinViewModel @Inject constructor() : ObservableViewModel() {
   val pinErrorMsg = ObservableField("")
   val confirmPinErrorMsg = ObservableField("")
+  var savedPin = ""
```

Comment:
We normally tend to avoid using `var`.

Update them to an observable field.
`val savedPin = ObservableField("")`
`val savedConfirmPin = ObservableField("")`

This will bring change in your other files to about how you set and fetch this value.

Reply:
Instead of `ObservableField("")` I was thinking of using `MutableLiveData("")` . It's similar to `ObservableField("")` but it saves the data during configuration changes. 
`ObservableField("")` was not saving the data. 

Now I am going to write the tests.

Reply:
I also resolved the error message disappearing issue